src/components/overlay/overlay.scss

```diff
@@ -0,0 +1,193 @@
+@use '../core/styles' as sbb;
+
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin
+@include sbb.host-component-properties;
+
+:host {
+  @include sbb.scrollbar-variables;
+
+  --sbb-overlay-padding-inline: var(--sbb-spacing-fixed-5x);
+  --sbb-overlay-color: var(--sbb-color-black);
+  --sbb-overlay-background-color: var(--sbb-color-milk);
+  --sbb-overlay-width: 100%;
+  --sbb-overlay-height: 100%;
+  --sbb-overlay-inset: 0 auto auto 0;
```

Comment:
needed for fullscreen? would 0 not be sufficient?

Reply:
It is already `0` when is open. This definition is for when it's closed and it hides on the top-left corner of the screen with 0x0 dimensions.